db/account.go

```diff
@@ -0,0 +1,82 @@
+package db
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+)
+
+const (
+	createAccount = `INSERT INTO account (id,balance,user_id,created_at) VALUES(?,?,?,SYSDATE())`
```

Comment:
dont use sysdate for setting time for created_at, updated_at columns

use time.now value